CMakeLists.txt

```diff
@@ -289,8 +310,15 @@ if(${STRINGZILLA_BUILD_TEST})
     endif()
   elseif(SZ_PLATFORM_ARM)
     # ARM specific backends
-    define_launcher(stringzilla_test_cpp20_arm_serial scripts/test.cpp 20 "armv8-a")
-    define_launcher(stringzilla_test_cpp20_arm_neon scripts/test.cpp 20 "armv8-a+simd")
+    if (MSVC)
+      define_launcher(stringzilla_test_cpp20_arm_serial scripts/test.cpp 20 "armv8.0")
+      target_compile_definitions(stringzilla_test_cpp20_arm_serial PRIVATE "SZ_USE_ARM_NEON=0")
+      target_compile_definitions(stringzilla_test_cpp20_arm_serial PRIVATE "SZ_USE_ARM_SVE=0")
+      define_launcher(stringzilla_test_cpp20_arm_neon scripts/test.cpp 20 "armv8.0")
+    else()
+      define_launcher(stringzilla_test_cpp20_arm_serial scripts/test.cpp 20 "armv8-a+nosimd")
+      define_launcher(stringzilla_test_cpp20_arm_neon scripts/test.cpp 20 "armv8-a+simd")
```

Comment:
I’m generally targeting at least 8.2 for NEON features.

Reply:
Okay, after a quick glance at the differences between 8 and 8.2, it seems that 8.2 added SVE support, so should we reflect this in the builds?